Update the version of the second `bandit` hook

mcdonnnj opened the pull request:
# <!-- Use the title to describe PR changes in the imperative mood --> #

## 🗣 Description ##

This pull request updates the `bandit (everything else)` hook to use the latest version of [bandit].
<!-- Describe the "what" of your changes in detail. -->
<!-- To avoid scope creep, limit changes to a single goal. -->

## 💭 Motivation and context ##

This hook was not kept in sync with updates to the `bandit (tests tree)` hook and is using an older version of [bandit].
<!-- Why is this change required? -->
<!-- What problem does this change solve? How did you solve it? -->
<!-- Mention any related issue(s) here using appropriate keywords such -->
<!-- as "closes" or "resolves" to auto-close them on merge. -->

## 🧪 Testing ##

Automated tests pass.
<!-- How did you test your changes? How could someone else test this PR? -->
<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the code, etc. -->

<!--
## 📷 Screenshots (if appropriate) ##

Uncomment this section if a screenshot is needed.

-->

## ✅ Pre-approval checklist ##

<!-- Remove any of the following that do not apply. -->
<!-- Draft PRs should have one or more unchecked boxes. -->
<!-- If you're unsure about any of these, don't hesitate to ask. -->
<!-- We're here to help! -->

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

[bandit]: https://github.com/PyCQA/bandit

mcdonnnj committed:
Update the version of the second bandit hook

The `bandit (everything else)` hook was not updated in sync with the
`bandit (tests tree)` hook and is using an older version of bandit.

dav3r commented on .pre-commit-config.yaml:
```diff
@@ -91,7 +91,7 @@ repos:
           - --config=.bandit.yml
   # Run bandit everything but tests directory
```
Can we clean up this comment as long as we're here?
```suggestion
  # Run bandit on everything except the tests directory
```

jsf9k committed:
Correct poor English in comment

Co-authored-by: dav3r <david.redmin@trio.dhs.gov>

mcdonnnj committed:
Add an explanatory comment to one of the bandit pre-commit hooks

Update the other bandit hook's explanatory comment so the language is
consistent.

dav3r reviewed (approved):
👍 👍
